frontend/app/component/election/ElectionStatusProgress.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+
+import { PollingStationStatus, PollingStationStatusEntry, useElection, useElectionStatus } from "@kiesraad/api";
+import { IconDot } from "@kiesraad/icon";
+import { Icon, PercentageAndColorClass, Progress, ProgressBar, ProgressBarColorClass } from "@kiesraad/ui";
+
+const statusCategories = ["in_progress", "unfinished", "definitive", "not_started"] as const;
+type StatusCategory = (typeof statusCategories)[number];
+
+const categoryTitle: Record<StatusCategory, string> = {
+  unfinished: "Niet afgeronde invoer",
+  in_progress: "Invoer bezig",
+  definitive: "Eerste invoer klaar", // TODO: in the future this changes to `Eerste en tweede invoer klaar`
+  not_started: "Werkvoorraad",
+};
+
+const categoryColorClass: Record<StatusCategory, ProgressBarColorClass> = {
+  unfinished: "unfinished",
+  in_progress: "in-progress",
+  definitive: "definitive",
+  not_started: "not-started",
+};
+
+function statusCount(entries: PollingStationStatusEntry[], status: PollingStationStatus): number {
+  return entries.filter((s) => s.status === status).length;
+}
+
+export function ElectionStatusProgress() {
+  const { pollingStations } = useElection();
+  const { statuses } = useElectionStatus();
```

Comment:
Do we need the `pollingStations` here? Because I'd expect `pollingStations.length` to be equal to `statuses.length`.

Reply:
Removed in e780f912277f30572ef6f5373ef491802cac4254